contracts/TokenUse.sol

```diff
@@ -76,6 +79,7 @@ contract TokenUse is DSAuth, ITokenUse, SettingIds {
 
     function createTokenUseOffer(uint256 _tokenId, uint256 _duration, uint256 _price, address _acceptedActivity, uint256 _fee) public {
         require(tokenId2UseStatus[_tokenId].user == address(0), "Token already in another use.");
+        require(IApostleBase(registry.addressOf(CONTRACT_MINER)).isReadyToBreed(_tokenId), "it is having baby. wait.");
```

Comment:
This sould be a general issue, meaning that a token cannot be used if it is currently in using. So where to find whether this token is in using, not only Apostle will face this issue.